.changeset/plenty-nails-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"import-map-overrides": patch
+---
+
+Add automation to publish package
```

Comment:
What is this file for? Im curious

Reply:
This is the changeset for this commit. You accumulate changesets file until you run `changeset version` which updates the version and the changelog and removes the changeset files.